UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -323,9 +316,20 @@ private void InitializeEnvironment()
                         key, resetParameters[key]
                     );
                 }
-
-                var pythonParameters = m_BrainBatcher.SendAcademyParameters(academyParameters);
-                Random.InitState(pythonParameters.Seed);
+                try
+                {
+                    var pythonParameters = m_BrainBatcher.SendAcademyParameters(academyParameters);
```

Comment:
What causes this to raise an exception? Please explain the logic around here more.

Reply:
I added a comment